lib/cookies.js

```diff
@@ -11,42 +11,32 @@
  *
  */
 
-var util = require('./util');
+var Cookies = require('js-cookie');
 
 function setCookie(name, value, expiresAt) {
-  var expiresText = '';
-  if (expiresAt) {
-    expiresText = ' expires=' + util.isoToUTCString(expiresAt) + ';';
-  }
-
-  var cookieText = name + '=' + value + '; path=/;' + expiresText;
-  setCookie._setDocumentCookie(cookieText);
-
-  return cookieText;
+  // The "expires" key with our cookie lib maps to "days until expired".
+  // Since we expect it to expire on a specific point in time, we
+  // subtract our current time to give us our "days until expired"
+  var future = new Date(expiresAt).getTime();
+  var now = new Date().getTime();
+
+  // Convert time in ms into days
+  // 1000 ms * 60 sec * 60 min * 24 hrs
+  var expires = Math.floor((future - now)/(1000*60*60*24));
```

Comment:
Why the floor/conversation to days?  This won't allow you to expire a cookie at a specific time (as we would need to do?).  The `js-cookies` lib can accept a date object for `expires`, why not use that?